README.adoc

```diff
@@ -83,16 +68,16 @@ fn main() {
         .parse::<SocketAddr>()
         .expect("Expect to parse address");
 
-    let client_addr = "127.0.0.1:8545"
-        .parse::<SocketAddr>()
-        .expect("Expect to parse client address");
-
-    emerald::start(&addr, &client_addr, None);
+    emerald::start(&addr, None);
```

Comment:
Emerald also needs the `base_path` for finding the place to store keyfiles, etc. Maybe there should be an extra parameter?